src/com/itschool/session6/homework/ex2/equalsMethod.java

```diff
@@ -0,0 +1,12 @@
+package com.itschool.session6.homework.ex2;
+
+public class equalsMethod {

```

Comment:
Rename the class to match the Java convention (class name should start with upper-case letter).